openshift/templates/issuer-web/issuer-web-deploy.openvp_rev.param

```diff
@@ -3,10 +3,10 @@
 # Component: .
 # Template File: ../openshift/templates/issuer-web/issuer-web-deploy.json
 #=========================================================
-NAME=issuer-web
-APP_NAME=issuer
+NAME=issuer-web-openvp-rev
+APP_NAME=issuer-openvp
 APP_GROUP=openvp
-SUFFIX=-openvp-rev
+SUFFIX=
```

Comment:
This change reverts the proper use of the suffix.

Reply:
This is correct for this component: it is a second frontend that needs to target the back-end service exposed by `openvp`. There is no way of using suffixed properly to deploy it AND have it grouped with the `openvp` apps and have the KNP for `openvp` let traffic through. By setting the parameters this way we can achieve all of the above without needing to spin-off a new template or new services.